tests/ConversationTranscriberTests.ts

```diff
@@ -386,3 +386,64 @@ test("Create Conversation with one channel audio (aligned)", (done: jest.DoneCal
             done.fail(error);
         });
 });
+test("Create Conversation and force disconnect", (done: jest.DoneCallback) => {
```

Comment:
Not sure why this and the other Transcription change are present here.. I did a rebase from master which I expected would resolve this, but these are not actually changes I've made

Reply:
This and some of the ServiceRecognizerBase changes were in my merged PR for the reconnect-fix branch.